src/random.h

```diff
@@ -67,8 +67,15 @@
  * Thread-safe.
  */
 void GetRandBytes(unsigned char* buf, int num) noexcept;
+/** Generate a random integer in the range [0..range). Precondition: range > 0 */
 uint64_t GetRand(uint64_t nMax) noexcept;
-std::chrono::microseconds GetRandMicros(std::chrono::microseconds duration_max) noexcept;
+/** Generate a random duration in the range [0..max). Precondition: max.count() > 0 */
+template <typename D>
+D GetRandDur(typename std::common_type<D>::type max) noexcept
```

Comment:
Would `GetRandDuration` or `GetRandomDuration` be too long? It would be clearer (and fwiw "Dur" means "hard" in French :smile:)

Reply:
Thanks. Fixed

Reply:
Thanks :)